# serializer version: 1
# name: test_pe_certify_users
  '''
  > about to resolve first_name and last_name for count=1 users.
  > only count=1 users have the necessary data to be resolved.
  ! could not find a match for pk=424242 error=PoleEmploiAPIBadResponse(code=R010)
  ! no match found either for pk=424242 when swapping last and first names exc=PoleEmploiAPIBadResponse(code=R010)
  > count=1 users have been examined.
  > count=0 users have been certified.
  > count=1 users could not be certified.
  > count=0 users have been swapped.
  
  '''
# ---
# name: test_pe_certify_users.1
  list([
    '> about to resolve first_name and last_name for count=1 users.',
    '> only count=1 users have the necessary data to be resolved.',
    '> certified user pk=424242 id_certifie=ruLuawDxNzERAFwxw6Na4V8A8UCXg6vXM_WKkx5j8UQ',
    '> count=1 users have been examined.',
    '> count=1 users have been certified.',
    '> count=0 users could not be certified.',
    '> count=0 users have been swapped.',
  ])
# ---
# name: test_pe_certify_users_with_swap
  list([
    '> about to resolve first_name and last_name for count=1 users.',
    '> only count=1 users have the necessary data to be resolved.',
    '! could not find a match for pk=424243 error=PoleEmploiAPIBadResponse(code=R010)',
    '> SWAP DETECTED! user pk=424243 id_certifie=ruLuawDxNzERAFwxw6Na4V8A8UCXg6vXM_WKkx5j8UQ',
    '> certified user pk=424243 id_certifie=ruLuawDxNzERAFwxw6Na4V8A8UCXg6vXM_WKkx5j8UQ',
    '> count=1 users have been examined.',
    '> count=1 users have been certified.',
    '> count=0 users could not be certified.',
    '> count=1 users have been swapped.',
  ])
# ---
